src/utils/objectToFormData.ts

```diff
@@ -0,0 +1,55 @@
+import fs from "fs"
+import FormData from "form-data"
+
+export type ObjectEntry = [string, any]
+export type ObjectEntries = Array<ObjectEntry>
+
+export function objectToFormData(object: object): FormData {
+  const form = new FormData()
+
+  for (const [key, value] of Object.entries(object)) {
+    if (value == null) continue
+
+    if (typeof value === "object") {
+      if (value instanceof fs.ReadStream) {
+        form.append(key, value)
+        continue
+      }
+
+      if (Array.isArray(value)) {
+        for (const valueElement of value) {
+          form.append(key, valueElement)
+        }
+
+        continue
+      }
```

Comment:
Nitpick: may this can be refactored to a recursive function for arrays that have more than one dimension, even I don't think we could need it in a request

Reply:
I've been trying to refactor this, but I'm not getting into any conclusion. Maybe a good change for making a refactoring excercise at Acid Labs time